crates/fuel-core/src/executor.rs

```diff
@@ -2754,7 +2753,7 @@ mod tests {
             .unwrap();
 
         // Assert the balance and state roots should be the same before and after execution.
-        let empty_state = Bytes32::from(*empty_sum_sha256());
+        let empty_state = [0; 32].into();
```

Comment:
[nit] We can use `MerkleTree::empty_root` instead of `[0; 32].into()`

Reply:
Unfortunately, we still need to provide the generic arguments to `MerkleTree`, so it turns into something like `MerkleTree<Table, StorageType<Table>>::empty_root()`, which is kinda ugly. That's why I've been using the empty in-memory trees for empty roots for now. But with the recent `fuel-vm` PR, we will be able to just use `sparse::empty_root()`.